frame/transaction-payment/src/lib.rs

```diff
@@ -304,6 +307,12 @@ pub mod pallet {
 		fn WeightToFee() -> Vec<WeightToFeeCoefficient<BalanceOf<T>>> {
 			T::WeightToFee::polynomial().to_vec()
 		}
+
+		#[allow(non_snake_case)]
```

Comment:
You should not need this since: https://github.com/paritytech/substrate/issues/8826

Is complete. Can you also update the note above and use the expected macro?

Reply:
2dd3822